packages/attribute-slicer/src/AttributeSlicer.ts

```diff
@@ -125,6 +125,23 @@ export class AttributeSlicer {
     private searchDebounce = SEARCH_DEBOUNCE;
 
     /**
+     * Whether or not to left align item text
+     */
+    private _leftAlignText = false;
+    get leftAlignText() {
+        return this._leftAlignText;
+    }
+    
+    /**
+     * Sets wheter or not to left align item text
+     */
+    set leftAlignText(value: boolean){
+        if (value !== this._leftAlignText){
+            this._leftAlignText = value;
```

Comment:
No, the rerendering is behind a `debounce`, so it *should* only actually be run once.